test/e2e/demo-plugin/Dockerfile

```diff
@@ -0,0 +1,11 @@
+#FROM golang:1.14.2 as build
```

Comment:
It would simplify everything to use COPY --from, but I'm on RHEL7 (with an older docker version) and I was not sure what was used in the build pipeline, so this was "emulated" in the Makefile currently (`make test/e2e/demo-plugin/demo-plugin`).

I'll remove it, it can be clean up later if everything supports docker multi stage builds.